utils.py

```diff
@@ -223,3 +223,12 @@ def get_profile_pic(bot, user_id, size):
     photo_file.download(custom_path=filename_jpg)
     filename_png = convert_jpg_to_png(filename_jpg, delete_jpg=True)
     return filename_png
+
+
+# Possessive apostrophe rule in english grammar
+def apostrophe(name):
```

Comment:
On `main.py`, the function was accepting `dixit_game.storyteller`, which is a `Player` object, not a string

Reply:
This is a fairly general procedure, it may be worthwhile to make it a method of the `Player` class? A general `Player.format_name(formal=False, posssessive=False, ...)` method, or something like that. It's not as clean as our current automatic string-conversion, in which `str(Player)` returns the name, though. @Megaleo

Reply:
I think a method like this one wouldn't hurt, and would aggregate these name changes to a single place.